lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -430,16 +433,16 @@ migrateManually tr proxy defaultFieldValues =
                 traceWith tr $ MsgManualMigrationNotNeeded field
 
             ColumnMissing -> do
-                let orig = "checkpoint"
-                let tmp = orig <> "_tmp"
-
-                -- 1. Add genesis_hash and genesis_start to the 'wallet' table.
                 [defaults] <- runSql conn $ select ["genesis_hash", "genesis_start"] orig
```

Comment:
Implicitly, this assumes that is the `genesis_hash` field does not exists in the `wallet` table, then it necessarily exists in the checkpoint table. This should be true for quite many releases back and indeed, works fine with the way the migration is written :+1: